### PR TITLE
Admins can claim prayers

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
@MrStonedOne added this to adminhelps and it's been useful, so I'd like to see something similar for prayers

### CODE REVIEW

**Comment by Dorsisdwarf** [comment]
Psst can you make this a button rather than attatched to SM. I like to use DirectNarrate so the prayer isn't being answered by YOLO SWAGGINS GOD OF BEER

**Comment by bgobandit** [comment]
too bad admins never touch prayers

**Comment by Anonmare** [comment]
>Pray
>Get called a faggot
okay...

**Comment by Niknakflak** [comment]
This isnt going to change anything personally.  Just make prayers display the current diety name like they do on vg when someone prays

**Comment by Shadowlight213** [comment]
^
It's really stupid that we can't see who the current diety is.

**Comment by KorPhaeron** [comment]
>This isnt going to change anything personally. Just make prayers display the current diety name like they do on vg when someone prays

I'd rather it just go back to "a voice in your head" so I can pretend to be anything from Space Jesus to the Syndicate depending on who I'm talking to

**Comment by Shadowlight213** [comment]
@KorPhaeron Actually do that. Change it back to a voice in your head. It's actually more immersion breaking with the diety response thing since having space Buddha reply to your prayer to the god of war for a tc trade is really retarded.

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -28,9 +28,11 @@
 		src << "Only administrators may use this command."
 		return
 
+	message_admins("[key_name_admin(src)] has started answering [key_name(M.key, 0, 0)]'s prayer.")
 	var/msg = input("Message:", text("Subtle PM to [M.key]")) as text
 
 	if (!msg)
+		message_admins("[key_name_admin(src)] decided not to answer [key_name(M.key, 0, 0)]'s prayer")
```
Comment (tkdrg) [inline code/modules/admin/verbs/randomverbs.dm]: these can just be `key_name(M, 0, 0)` instead

**Comment by Chocobro** [comment]
>bgobandit
>too bad admins never touch prayers

press SC to be good at prayers

**Pull request merged**